application/deed/searchdeed/views.py

```diff
@@ -61,6 +62,7 @@ def do_search_deed_search(form):
         deed_data = lookup_deed(deed_token['deed_token'])
 
     if deed_data is not None:
+        deed_data["deed"]["address"] = format_address_string(deed_data["deed"]["address"])
```

Comment:
I know the naming of the 'address' was not part of this story but i dislike the naming convention for this. Please Change to 'property_address' to remove any ambiguity. Can you update the swagger to reflect this and i'm sure it will break a few unit tests :)